### PR TITLE
Fix CORB Blocking (pt. 2)

### PR DESCRIPTION
**Pull request by digitaluniverse**
Fix CORB Blocking by calling all the requests in the background.js

### CODE REVIEW

**Comment by tejashah88** [comment]
@digitaluniverse I see that you've made 2 PR for the problem. I do request that in the future, all fixes related to a single problem are in one PR, to avoid confusion for both of us.

File: src/inject.js [hunk 1]
```diff
@@ -207,11 +227,13 @@ var debugOutput = stuff => { console.log(stuff); return stuff; };
 var debugJsonOutput = stuff => { console.log(JSON.stringify(stuff, null, 2)); return stuff; };
 
 function searchDefinitions(query, callback) {
-  fetch(searchStrictUrl(query))
+  console.log('This function is being called');
+//fetch(searchStrictUrl(query))
```
Comment (tejashah88) [inline src/inject.js]: cleanup here

File: src/inject.js [hunk 2]
```diff
@@ -233,6 +255,7 @@ function searchDefinitions(query, callback) {
     // return the first answer back to the background page
     .then(finalAnswer => callback({ success: finalAnswer }))
     .catch(err => {
+      console.log('There is an error');
```
Comment (tejashah88) [inline src/inject.js]: cleanup here as well

Comment (digitaluniverse) [reply src/inject.js]: Oops Thanks for catching that

**Comment by digitaluniverse** [comment]
the second commit should have changes

**Comment by tejashah88** [comment]
Perfect! Merging...

**Pull request merged**